addons/zeus/functions/fnc_moduleStaticWeaponPlace.sqf

```diff
@@ -0,0 +1,65 @@
+/*
+ * Author: PabstMirror
+ * Adapted from BIS_fnc_packStaticWeapon (Dean "Rocket" Hall, reworked by Killzone_Kid)
+ * Place on a unit in a group carrying static weapon backpacks to make them setup the weapon
+ *
+ * Arguments:
+ * 0: Module logic <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [LOGIC] call ace_zeus_fnc_moduleStaticWeaponPlace
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+params ["_logic"];
+if (!local _logic) exitWith {};
+
+if (canSuspend) exitWith {[FUNC(moduleStaticWeaponPlace), _this] call CBA_fnc_directCall;};
+
+// Validate the module target
+private _unit = effectiveCommander (attachedTo _logic);
+private _group = group _unit;
+TRACE_3("moduleStaticWeaponPlace placed",_unit,typeOf _unit,typeOf _logic);
+
+deleteVehicle _logic; // cleanup logic now, we just needed it to get the attached unit
+
+if (isNull _unit) exitWith { [LSTRING(NothingSelected)] call FUNC(showMessage); };
+if (!alive _unit) exitWith { [LSTRING(OnlyAlive)] call FUNC(showMessage); };
+if (!(_unit isKindOf "CAManBase")) then { [LSTRING(OnlyInfantry)] call FUNC(showMessage); };
+if (({[_x] call EFUNC(common,isPlayer)} count units _group) != 0) exitWith { ["str_a3_cfgvehicles_moduleremotecontrol_f_errorPlayer"] call FUNC(showMessage); };
+
+private _gunner = objNull;
+private _assistant = objNull;
+private _availableUnits = [_unit]; // try to seach attached unit first
+_availableUnits append ((units _group) select {(!isNull _x) && {_unit != _x}});
+_availableUnits = _availableUnits select {([_x] call EFUNC(common,isAwake)) && {(vehicle _x) == _x}};
+
+{
+    if ((unitBackpack _x) isKindOf "Weapon_Bag_Base") exitWith {_gunner = _x};
+} forEach _availableUnits;
+
+if (isNull _gunner) exitWith { [localize LSTRING(StaticWeapon_NotCarrying)] call FUNC(showMessage); };
+private _cfgBase = configFile >> "CfgVehicles" >> (backpack _gunner) >> "assembleInfo" >> "base";
+private _compatibleBases = if (isText _cfgBase) then {[getText _cfgBase]} else {getArray _cfgBase};
+{
+    private _xUnit = _x;
+    if (({(unitBackpack _xUnit) isKindOf _x} count _compatibleBases) > 0) exitWith {_assistant = _xUnit};
+} forEach _availableUnits;
```

Comment:
For the assistant checks, it could potentially select a player, should not is player be checked?

Reply:
handled by line 34, if any member of the group is a player it will exit